app/results.js

```diff
@@ -73,7 +73,7 @@ export function decodedToVariantOutputOnlyBox(
   forReact = forReact && forReact.length ? forReact : <Placeholder />
   const variantBox = (
     <VariantOutputOnlyBox
-      key={idx}
+      key={decodedVariant.key || decodedVariant.label}
```

Comment:
Nevidím žádný reálný přínos této změny. Původní verze `key={idx}` je pro mě srozumitelnější.